src/commands/functions/functions-create.js

```diff
@@ -582,11 +663,12 @@ const ensureFunctionPathIsOk = function (functionsDir, name) {
  * @param {import('../base-command').BaseCommand} command
  */
 const functionsCreate = async (name, options, command) => {
-  const functionsDir = await ensureFunctionDirExists(command)
+  const isEdgeFunc = (await selectTypeOfFunc()) === 'Edge Function (Deno)'
```

Comment:
With `inquirer`, you can make `choices` an array of objects with a `name` and `value` properties. While `name` is presented to the user, `value` is what'll be added to the answers object. This way you could check for a value like `edgeFunction` and not the human-friendly, which we might want to change.

That was a bit difficult to explain, but hopefully it makes sense?

Reply:
I think I see what you mean, setting a `value` property to `edgeFunction` is much cleaner than looking for the human-readable version, and if the prompt changes in the future, it'll only have to be changed in one place. 

I'm just trying to wrap my head around this in conjunction with your other comment before:

> Rather than a isEdgeFunc boolean, do you think we could we have a type enum variable containing serverless or edge? I feel that could be extended to contain other types in the future.

Going to try to walk through this to check my understanding:

- If we create an enum with the types of functions (currently `serverless` and `edge`), rather than a boolean, we have more flexibility to add other types of functions in the future
- Rather than passing around `isEdgeFunc`, we'd pass around something like `funcType`, which will be `serverless` or `edge`
- When we prompt users to choose their type of function using `inquirer`, the values should be the same as the enum values, because that's how the variable will get set.

Does that make sense?

Reply:
That totally makes sense! 💯